idex.html

```diff
@@ -11,208 +11,230 @@
                 </ul>
-            </li>
-            <li>
-                <div class="knowledge__title">Frameworks<span class="material-symbols-outlined">
-                        chevron_right
-                    </span>
-                </div>
-            </li>
-            <hr>
-            <li>
-                <div class="knowledge__title">Skills<span class="material-symbols-outlined">
-                        chevron_right
-                    </span>
-                </div>
-            </li>
-            <hr>
-        </ul>
-
-    </section>
-    <!-- Footer -->
-    <footer class="footer">
-        <h3 class="footer__title">Contact me</h3>
-        <p class="footer__paragraph">If you have an application you are interested in developing, a feature that you
-            need built or a project
-            that needs coding. I’d love to help with it </p>
-        <form class="footer__form" action="https://formspree.io/f/xwkjeknw" method="post">
-            <ul class="footer__ul">
+                <button class="about__button">Get my resume</button>
+            </div>
+            <ul class="knowledge">
                 <li>
-                    <label for=""></label>
-                    <input class="footer__input" type="text" id="name" name="user_name" placeholder="Your name"
-                        maxlength="30" required />
+                    <div class="knowledge__titles">Languages<span class="material-symbols-outlined">
+                            expand_more
+                        </span>
+                    </div>
+                    <ul class="about__info--technologies">
+                        <li><img class="about__images" src="./images/img-languages-js.jpg" alt="js icon">JavaScript</li>
+                        <li><img class="about__images" src="./images/img-languages-html.jpg" alt="html icon">HTML</li>
+                        <li><img class="about__images" src="./images/img-languages-css.jpg" alt="css icon">CSS</li>
+                    </ul>
                 </li>
                 <li>
-                    <label for=""></label>
-                    <input class="footer__input" type="email" id="mail" name="user_email" placeholder="Your email"
-                        required />
+                    <div class="knowledge__titles">Frameworks<span class="material-symbols-outlined">
+                            chevron_right
+                        </span>
+                    </div>
                 </li>
+                <hr>
                 <li>
-                    <label for=""></label>
-                    <textarea class="footer__input footer__textarea" id="msg" name="user_message"
-                        placeholder="Write your message here" maxlength="500" required></textarea>
+                    <div class="knowledge__titles">Skills<span class="material-symbols-outlined">
+                            chevron_right
+                        </span>
```

Comment:
- [ ] kindly align the image and the title of the languages to the left to match the figma design

![image](https://user-images.githubusercontent.com/89282221/213803092-5ce31d94-f725-4412-9ff9-60dd7c38768f.png)

figma: 

![image](https://user-images.githubusercontent.com/89282221/213803138-a0584191-35b1-4824-a56f-7ff2a03df032.png)